endpoints/openrtb2/auction.go

```diff
@@ -879,6 +879,12 @@ func setImpsImplicitly(httpReq *http.Request, imps []openrtb.Imp) {
 	}
 }
 
+var errBadIncomingRequest = fmt.Errorf("Invalid JSON in Incoming Request")
+var errBadStoredRequest = fmt.Errorf("Invalid JSON in Stored Request")
+var errBadIncomingImp = fmt.Errorf("Invalid JSON in Incoming Imp Request")
+var errBadStoredImp = fmt.Errorf("Invalid JSON in Stored Imp Request")
+var errBadDefaultRequest = fmt.Errorf("Invalid Default Request")
```

Comment:
Probably want to add that the issue is with the server config, not anything related to the request.

`Invalid JSON in Default Request: The server has been configured with an invalid default for all requests.`